variables.tf

```diff
@@ -226,6 +202,12 @@ variable "user_data" {
   default     = null
 }
 
+variable "install_ssm_agent" {
```

Comment:
If we have this variable, do we need two ssm scripts?

Reply:
One has now been now removed